### PR TITLE
chore: migrate to ES6 module

### PR DESCRIPTION
**Pull request by jobo322**

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current',
+        },
+      },
+    ],
+  ],
+};
```
Comment (lpatiny) [inline babel.config.js]: Babel config should only contain: https://github.com/mljs/peak-shape-generator/blob/master/babel.config.js

File: package.json [hunk 2]
```diff
@@ -29,6 +29,8 @@
   },
   "homepage": "https://github.com/mljs/matrix-peaks-finder#readme",
   "devDependencies": {
+    "@babel/preset-env": "^7.11.5",
+    "babel-jest": "^26.5.2",
```
Comment (lpatiny) [inline package.json]: and package should only contain (related to babel): https://github.com/mljs/peak-shape-generator/blob/master/package.json#L41

This is used by jest to transpile the code before testing it.

File: src/test/index.test.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-let FS = require('fs');
+import * as FS from 'fs';
```
Comment (lpatiny) [inline src/test/index.test.js]: You may directly import the method you will use like `{readFileSync}`

**Review by lpatiny — CHANGES_REQUESTED** [review]

File: src/test/index.test.js (hunk 3 above)
Comment (jobo322) [reply src/test/index.test.js]: it's done

File: package.json (hunk 2 above)
Comment (jobo322) [reply package.json]: it's done

File: babel.config.js (hunk 1 above)
Comment (jobo322) [reply babel.config.js]: it's done

File: package.json (hunk 2 above)
Comment (lpatiny) [reply package.json]: babel-jest should not be there seems to me

**Pull request merged**